File: server/Models/Receipt.cs
namespace palletShop.Models;
public class Receipt
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Buyer { get; set; }
    public string Street { get; set; }
    public string City { get; set; }
    public string State1 { get; set; }
    public string Zip { get; set; }
    public int Total { get; set; }
    public Account Creator { get; set; }
    public string CreatorId { get; set; }
    public int PurchaseId { get; set; }
}

File: client/src/models/Receipt.js
export class Receipt{
    constructor(data){
        this.id = data.id
        this.createdAt = new Date(data.createdAt).toLocaleDateString()
        this.updatedAt = new Date(data.updatedAt).toLocaleDateString()
        this.purchaseId = data.purchaseId
        this.buyer = data.buyer
        this.street = data.street
        this.city = data.city
        this.state1 = data.state1
        this.zip = data.zip
        this.total = data.total
        this.creator =data.creator
        this.creatorId = data.creatorId
    }
}